src/main/scala/cromwell/server/WorkflowManagerSystem.scala

```diff
@@ -0,0 +1,15 @@
+package cromwell.server
+
+import akka.actor.ActorSystem
+import cromwell.engine.WorkflowManagerActor
+
+trait WorkflowManagerSystem {
+  val systemName = "cromwell-system"
+  val actorSystem = ActorSystem(systemName)
+
+  actorSystem.registerOnTermination({
+    actorSystem.log.info(s"$systemName shutting down")
+  })
```

Comment:
In this call-by-name I think the curlies could be removed and this can become a single line, or is that considered bad form?